activerecord/lib/active_record/querying.rb

```diff
@@ -51,7 +51,7 @@ def find_by_sql(sql, binds = [], preparable: nil, &block)
       _load_from_sql(_query_by_sql(sql, binds, preparable: preparable), &block)
     end
 
-    # Same as <tt>#find_by_sql</tt> but perform the query asynchronously and returns an ActiveRecord::Promise
+    # Same as <tt>#find_by_sql</tt> but perform the query asynchronously and returns an +ActiveRecord::Promise+.
```

Comment:
I don't think we should be adding these `+`. By omitting these we are currently linking to `ActiveRecord::Promise`, which we lose if we add the `+`:

![image](https://github.com/rails/rails/assets/6014046/6b7a0a61-63bb-4fca-af0c-0d68c9b09c3b)

Adding the `.` seems like a good change though

Reply:
Sure, I've updated this PR. Thank you!